cost_estimate.go

```diff
@@ -33,13 +33,14 @@ type costEstimates struct {
 // CostEstimateStatus represents a costEstimate state.
 type CostEstimateStatus string
 
-//List all available costEstimate statuses.
+// List all available costEstimate statuses.
 const (
-	CostEstimateCanceled CostEstimateStatus = "canceled"
-	CostEstimateErrored  CostEstimateStatus = "errored"
-	CostEstimateFinished CostEstimateStatus = "finished"
-	CostEstimatePending  CostEstimateStatus = "pending"
-	CostEstimateQueued   CostEstimateStatus = "queued"
+	CostEstimateCanceled              CostEstimateStatus = "canceled"
+	CostEstimateErrored               CostEstimateStatus = "errored"
+	CostEstimateFinished              CostEstimateStatus = "finished"
+	CostEstimatePending               CostEstimateStatus = "pending"
+	CostEstimateQueued                CostEstimateStatus = "queued"
+	CostEstimateSkippedDueToTargeting CostEstimateStatus = "skipped_due_to_targeting"
```

Comment:
In the CostEstimateStatusTimestamps below, we could potentially also add SkippedDueToTargetingAt since that is a new valid status. That said, maybe with how "skipped_due_to_targeting" works, it would be a totally useless addition - I'm not sure

Reply:
Great catch! I don't see it being used in Terraform, but maybe it's used elsewhere? Might as well add it now so someone else doesn't have to later.

Reply:
nice - lgtm!